redaxo/src/core/lib/fragment.php

```diff
@@ -110,7 +110,7 @@ public function parse($filename)
                 $content = rex_timer::measure('Fragment: '.$filename, function () use ($fragment) {
                     ob_start();
                     require $fragment;
-                    return ob_get_clean();
+                    return ob_get_clean() || '';
```

Comment:
hast du zuletzt javascript programmiert? :)
In PHP kommt da ja nun ein boolscher Wert raus..

Reply:
hehe, stimmt.. ;-)